app/actions/Onboard.js

```diff
@@ -16,53 +18,42 @@ export function checkTezsterCliAction() {
       type: 'TEZSTER_CLI_PENDING',
       payload: false,
     });
-    setTimeout(() => {
-      if (process.platform.split('win').length > 1) {
-        const ls = spawn(
-          'cmd.exe',
-          [
-            '/c',
-            `powershell.exe tezster get-balance ${config.identities[0].pkh}`,
-          ],
-          { detached: false }
-        );
-        ls.stdout.on('data', (data) => {
+    RpcRequest.checkNodeStatus(url)
+      .then((res) => {
+        if (res.protocol.startsWith('PsCARTHAG')) {
+          return setTimeout(() => {
+            dispatch({
+              type: 'TEZSTER_SHOW_STOP_NODES',
+              payload: true,
+            });
+            return dispatch({
+              type: 'TEZSTER_CLI_SUCCESS',
```

Comment:
remove reference to CLI. You're not downloading or setting up CLI.
It'll create unnecessary confusion in the codebase.